src/test/java/org/sagebionetworks/bridge/sdk/integration/UploadTest.java

```diff
@@ -1,125 +1,217 @@
-        user = testUser.getSession().getUserClient();
+        UploadSchema iosSurveySchema = null;
+        try {
+            iosSurveySchema = researcherClient.getUploadSchemaById("ios-survey");
+        } catch (EntityNotFoundException ex) {
+            // no-op
+        }
+        if (iosSurveySchema == null) {
+            iosSurveySchema = new UploadSchema.Builder()
+                    .withSchemaId("ios-survey")
+                    .withName("iOS Survey Response")
+                    .withFieldDefinitions(ImmutableList.of(
+                            makeUploadFieldDef("answers", UploadFieldType.ATTACHMENT_JSON_TABLE),
+                            makeUploadFieldDef("item", UploadFieldType.STRING),
+                            makeUploadFieldDef("taskRunId", UploadFieldType.STRING)))
+                    .build();
+            researcherClient.createOrUpdateUploadSchema(iosSurveySchema);
+        }
 
-        createTextFile();
+        UploadSchema jsonDataSchema = null;
+        try {
+            jsonDataSchema = researcherClient.getUploadSchemaById("upload-test-json-data");
+        } catch (EntityNotFoundException ex) {
+            // no-op
+        }
+        if (jsonDataSchema == null) {
+            jsonDataSchema = new UploadSchema.Builder()
+                    .withSchemaId("upload-test-json-data")
+                    .withName("Upload Test JSON Data")
+                    .withFieldDefinitions(ImmutableList.of(
+                            makeUploadFieldDef("string.json.string", UploadFieldType.STRING),
+                            makeUploadFieldDef("blob.json.blob", UploadFieldType.ATTACHMENT_JSON_BLOB)))
+                    .build();
+            researcherClient.createOrUpdateUploadSchema(jsonDataSchema);
+        }
+
+        UploadSchema nonJsonSchema = null;
+        try {
+            nonJsonSchema = researcherClient.getUploadSchemaById("upload-test-non-json");
+        } catch (EntityNotFoundException ex) {
+            // no-op
+        }
+        if (nonJsonSchema == null) {
+            nonJsonSchema = new UploadSchema.Builder()
+                    .withSchemaId("upload-test-non-json")
+                    // in the current version, non-JSON data matches by name, not by ID, so for testing purposes,
+                    // keep schema and name the same
+                    .withName("upload-test-non-json")
+                    .withFieldDefinitions(ImmutableList.of(
+                            makeUploadFieldDef("nonJson.txt", UploadFieldType.ATTACHMENT_BLOB),
+                            makeUploadFieldDef("jsonFile.json", UploadFieldType.ATTACHMENT_JSON_BLOB)))
+                    .build();
+            researcherClient.createOrUpdateUploadSchema(nonJsonSchema);
+        }
     }
 
-    @After
-    public void after() {
-        testUser.signOutAndDeleteUser();
-        deleteTextFile();
+    @AfterClass
+    public static void afterClass() {
+        if (researcher != null) {
+            try {
```

Comment:
The idea was a failured in deleting the researcher won't prevent us from deleting the user. I changed this to delete the researcher and user in separate AfterClass methods.